src/components/Filter/Filter.js

```diff
@@ -74,6 +74,7 @@ export default class Filter extends Component {
               source={tags}
               onSelect={this.onTagSelect}
               clickedTag={clickedTag}
+              clearButton={true}
```

Comment:
Could be shortened to `clearButton` (in all cases in this file).

Reply:
Done!